appinventor/docs/reference/components/animation.html

```diff
@@ -633,13 +633,17 @@ <h3>Methods</h3>
                  <dl>
                    <dt><code>Clear()</code></dt>
                    <dd>Clears anything drawn on this Canvas but not any background color or image.</dd>
+                   <dt><code>DrawArc(number left, number top, number right, number bottom, number startAngle, number sweepAngle, boolean useCenter, boolean fill)</code></dt>
+                   <dd>Draw an arc on Canvas, by drawing a arc from a specified oval (specified by left, top, right &amp; bottom), start angle is 0 when heading to the right, and increase when rotate clockwise.</dd>
                    <dt><code>DrawCircle(number x, number y, number r)</code></dt>
                    <dd>Draws a circle (filled in) at the given coordinates on the canvas, with the
                   given radius.</dd>
                    <dt><code>DrawLine(number x1, number y1, number x2, number y2)</code></dt>
                    <dd>Draws a line between the given coordinates on the canvas.</dd>
                    <dt><code>DrawPoint(number x, number y)</code></dt>
                    <dd>Draws a point at the given coordinates on the canvas.</dd>
+                   <dt><code>DrawShape(list pointList, boolean fill)</code></dt>
+                   <dd>Draws a shape on the canvas, point list should be a list contains sub-lists with two number which represents a coordinate</dd>
```

Comment:
Include an example of the point list here as well.

Reply:
Question: When you locally test your Canvas.DrawArc feature, in the advanced Any, do you see the "for component" in the block?
Look at the Canvas.Clear - it shows "for component"
Now look at Canvas.DrawArc - it doesn't show "for component" (NOTE: I built this using different builder codebase).

![image](https://user-images.githubusercontent.com/36113589/48194273-8c760600-e31a-11e8-96e5-5d3fbdac9489.png)

Reply:
@Amerkashi we've had this in Kodular for a while and it works fine:
![screenshot_20181108-115120 2](https://user-images.githubusercontent.com/25845449/48197226-0fb04f80-e34d-11e8-88d0-ce6995481ac8.png)